docs/database/updates/2016-12-22_1-schema.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE `audios` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `sentence_id` int(11) NOT NULL,
+  `user_id` int(11) DEFAULT NULL,
+  `external` varchar(500) DEFAULT NULL,
+  `created` datetime NOT NULL,
+  `modified` datetime NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `sentence_id` (`sentence_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+ALTER TABLE users
+  ADD `audio_license` varchar(50) CHARACTER SET utf8 COLLATE utf8_unicode_ci DEFAULT NULL,
+  ADD `audio_attribution_url` varchar(255) CHARACTER SET utf8 COLLATE utf8_unicode_ci DEFAULT NULL;
+
+INSERT INTO audios
+(sentence_id, created, modified)
+SELECT
+  id as sentence_id,
+  0 as created,
+  0 as modified
+FROM sentences
+WHERE hasaudio != 'no';
+
+ALTER TABLE sentences DROP hasaudio;
```

Comment:
> adapting this PR like you said sounds like more work than updating the deduplication script

I guess it depends from which perspective :) From my perspective, I much feel less confident touching the deduplication script because I haven't worked on it. So I would spend more time overall, because of the learning curve and because I would be more paranoid to mess up something.

> keeping the hasaudio field will make it much harder for us to notice that something broke

That's true. I guess we can find a compromise with:
```
ALTER TABLE sentences CHANGE COLUMN `hasaudio` `hasaudio_old`
enum('no','from_users','shtooka') NOT NULL DEFAULT 'no';
```
It's easier/faster to revert this, if we ever need to revert. It happened to me on the dev instance, that I after I deployed your PR, I wanted to test another PR and needed to revert back but the column was deleted.